rubygem-bastion/rubygem-bastion.spec

```diff
@@ -57,8 +55,6 @@ This package contains documentation for rubygem-%{gem_name}.
 Summary:   Provides asset compilation dependencies for %{scl_prefix}rubygem-%{gem_name}
 BuildArch: noarch
 Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
-Requires:  %{?scl_prefix}rubygem(less-rails) >= 2.5.0
-Requires:  %{?scl_prefix}rubygem(less-rails) < 2.6
```

Comment:
Please also add a commit to remove this package (just the dir, tito.props entries and comps/).